### PR TITLE
Adding Delete CCAP Pair page

### PR DESCRIPTION
**Pull request by EstebanSkyline**
Adding Delete CCAP Pair page and updating creating CCAP Page

### CODE REVIEW

File: user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md [hunk 1]
```diff
@@ -152,28 +128,35 @@ The elements will be created with the following specifications:
...
 1. Click *Create Bulk*.
 
-The elements will be created with the specifications in the CSV file of each row.
+   The elements will be created with the specifications in the CSV file of each row.
 
-Here is an example of the CSV file with the mandatory headers and the content of each row.
+   > [!NOTE]
+   >
+   > - Each time the script is executed, it will attempt to create five elements. If there are fewer than five elements left to create, it will create as many as possible without any issues. After the first set of elements has been created, you can execute the script again without modifying the CSV file, and it will attempt to create the next set of five elements.
+   > - Click the *Cancel* button in any window to close the Automation script at any time.
```
Comment (miguel-slc) [inline user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md:156]: Maybe we can include *not as a note* that the script will provision devices in bulk of 5 elements.

Comment (EstebanSkyline) [reply user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md:161]: Information included not as a note

File: user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md [hunk 2]
```diff
@@ -4,7 +4,10 @@ uid: I-DOCSIS_Create_CCAP_CM_pair
 
 # Creating a new CCAP/CM pair
 
-To create CCAP/CM pairs, you can either create a single CCAP/CM pair, or create elements in bulk from a CSV file.
+To create CCAP/CM pairs, you can either create a single CCAP/CM pair at a time, or create elements in bulk from a CSV file.
+
+> [!NOTE]
+> The creation of HarmonicOS pairs is not fully supported yet. While the elements will be created, some additional configuration will be missing.
```
[the remark this answers is not in the document]
Comment (EstebanSkyline) [reply user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md]: Note has been changed to > The creation of CCAP pairs using the connector Harmonic CableOS is currently not supported.

**Comment by miguel-slc** [comment]
Updates looks ok!

**Review by miguel-slc — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**